.travis.yml

```diff
@@ -8,9 +8,22 @@ before_install:
 - yarn global add codecov
 
 script:
+- node -e "assert.ok(!/^(?:[67]|8\.[01])\./.test(require('@angular/compiler').VERSION.full))"
 - yarn run lint
 - yarn run test
 
+- yarn upgrade @angular/compiler@8.1
+- node -e "assert.equal(require('@angular/compiler').VERSION.full.slice(0, 4), '8.1.')"
```

Comment:
You could specify the actual version (e.g., `yarn upgrade @angular/compiler@8.0.0`) to simplify the assertion:

```js
assert.equal(require('@angular/compiler').VERSION.full, '8.0.0')
```

Reply:
Done.